core/platform/models.py

```diff
@@ -33,139 +33,254 @@ def import_models(cls):
-    def import_models(cls, model_names):
-
-        returned_models = []
-        for name in model_names:
-            if name == NAMES.activity:
-                from core.storage.activity import gae_models as activity_models
-                returned_models.append(activity_models)
-            elif name == NAMES.base_model:
-                from core.storage.base_model import gae_models as base_models
-                returned_models.append(base_models)
-            elif name == NAMES.classifier:
-                from core.storage.classifier import gae_models as classifier_models # pylint: disable=line-too-long
-                returned_models.append(classifier_models)
-            elif name == NAMES.collection:
-                from core.storage.collection import gae_models as collection_models # pylint: disable=line-too-long
-                returned_models.append(collection_models)
-            elif name == NAMES.config:
-                from core.storage.config import gae_models as config_models
-                returned_models.append(config_models)
-            elif name == NAMES.email:
-                from core.storage.email import gae_models as email_models
-                returned_models.append(email_models)
-            elif name == NAMES.exploration:
-                from core.storage.exploration import gae_models as exp_models
-                returned_models.append(exp_models)
-            elif name == NAMES.feedback:
-                from core.storage.feedback import gae_models as feedback_models
-                returned_models.append(feedback_models)
-            elif name == NAMES.file:
-                from core.storage.file import gae_models as file_models
-                returned_models.append(file_models)
-            elif name == NAMES.job:
-                from core.storage.job import gae_models as job_models
-                returned_models.append(job_models)
-            elif name == NAMES.recommendations:
-                from core.storage.recommendations import gae_models as recommendations_models # pylint: disable=line-too-long
-                returned_models.append(recommendations_models)
-            elif name == NAMES.statistics:
-                from core.storage.statistics import gae_models as statistics_models # pylint: disable=line-too-long
-                returned_models.append(statistics_models)
-            elif name == NAMES.user:
-                from core.storage.user import gae_models as user_models
-                returned_models.append(user_models)
-            else:
-                raise Exception('Invalid model name: %s' % name)
-
-        return tuple(returned_models)
+	"""
+	Google App Engine Model.
+	Super model of the Registry Model and implements all the methods of Registry.
+	"""
+	@classmethod
+	def import_models(cls, model_names):
+		"""
+		Method that imports the required storage models from their particular modules.
+		Appends to an empty list according to If-Elif branching.
```

Comment:
This is an implementation detail; a caller would not care about the internal mechanism. Think about this from the perspective of someone "outside" the code in this file -- you want to give them a clear, succinct summary of what this method does.

So, your one-line summary docstring could be: "Imports and returns the storage models listed in model_names."